File: day048/homework/code/codewars9.py
# https://www.codewars.com/kata/56269eb78ad2e4ced1000013/train/python

def find_next_square(sq):
    root = int(sq**0.5)
    if root * root == sq:      
        return (root + 1) ** 2 
    else:
        return -1           

File: day068/homework/code/codewars9.js
// https://www.codewars.com/kata/585d7d5adb20cf33cb000235/train/python

def find_uniq(arr):
    a, b, c = arr[0], arr[1], arr[2]
    common = a if a == b or a == c else b
    for num in arr:
        if num != common:
            return num
